core/logger/logger.go

```diff
@@ -143,7 +144,12 @@ func newLogger(logLevel zapcore.Level, dir string, jsonConsole bool, toDisk bool
 	cfg.Level.SetLevel(logLevel)
 	l, err := newZapLogger(cfg)
 	if err != nil {
-		log.Fatal(err)
+		debug.PrintStack()
+		// fmt.Println("BALLS")
+		debug.PrintStack()
+		// time.Sleep(1 * time.Second)
+		// panic("BALLS")
+		log.Fatal(err.Error())
```

Comment:
Crashes here on invalid log file but doesn't print any of the stacks or the error?!

Reply:
Do we need those comments or we can safely remove them?